### DISCUSSION TITLE
How to disable the LED RGB bars? G15 513QM

### DISCUSSION THREAD

**Discussion by sadjesse** [seerge/g-helper#303] [Q&A]
First, I just wanted to say that I love this app! Great job! I love it :-)

I was wondering if there is a possibility to disable the lightbars and just have the keyboard have lights?
Thank you in advanced

Kind regards

**Comment by pgrabowski1** [comment]
Hi, I've got a G15 G513RW model. GHelper v0.55 added a bugfix for a situation when disabling keyboard backlight would disable LED bar as well. https://github.com/seerge/g-helper/issues/273
Maybe this bug will be some kind of a hint on what to do in order to enable/disable LED bard independently of keyboard backlight.

**Comment by WYLDrhett** [comment]
Yes I am wondering the same thing. I can turn the keyboard on and off but none of the other rgb settings, especially on boot and shutdown. Everytime I boot or shutdown my computer my laptop looks like a spacecraft lighting up haha. The only thing that works is turning the keyboard lights off for boot and shutdown but not the other stuff. Is there a way we could control the other lighting effects as well?

**Comment by seerge** [comment]
@sadjesse , @pgrabowski1 , @WYLDrhett 

Try this build, I have made so keyboard backlight checkboxes will also affect all (known) other lights in system. For example Awake checkbox should turn off all lights at all :

[GHelper.zip](https://github.com/seerge/g-helper/files/11404606/GHelper.zip)

![Screenshot 2023-05-05 105004](https://user-images.githubusercontent.com/5920850/236415566-40c5f9d4-9e41-4587-9d98-e5bd03d9e876.png)

**Reply by pgrabowski1** [reply]
Hi. This version indeed allows to control keyboard backlight and LED lightbar at the same time. 

Is it possible to control the LED bar independently of keyboard brightness? In other words to add a separate button that controls only LED bar brightness without affecting keyboard backlight.

**Reply by seerge** [reply]
@pgrabowski1 yeah, cause i linked all sections to one checkbox, i can try to split them.

**Comment by WYLDrhett** [comment]
@seerge Yes, it works! Thank you so much. This has saved me from much frustration because armoury crate sucks! Thank you so much!
